### ISSUE TITLE
Clear or Rest Votes

### ISSUE THREAD

**Issue by henry-pozzetta** [tatablack/leaner-coffee-powerup#51]
As a facilitator of Lean Coffee events, using the Leaner Coffee Power-up in Trello, I need the ability to clear or reset all votes on a board, either at one time, or individually by column or card, so that I can persistently use the same board over multiple Lean Coffee sessions with the same audience.

**Comment by tatablack** [comment]
Makes sense. I may already have some code to this effect, which I use while debugging. Shouldn't be too hard to make it a feature. I'll see if I can get some time to work on it this weekend.

**Comment by amandavarella** [comment]
That would be great. I also miss this. To run a second time we end up having to add our faces to vote.

**Comment by tatablack** [comment]
This shows how long it's been since I worked on this project... this feature was already partially implemented in #14, by adding a "Clear all votes from this list" menu item to the list menu (i.e. by "column", @henry-pozzetta).

<img width="285" alt="Screenshot 2022-03-25 at 01 43 20" src="https://user-images.githubusercontent.com/1095044/160033378-6fa8c6f3-699c-4149-8d6b-419702515462.png">

I may still look into adding this at the board level, assuming I can find a UX that's not too intrusive. I'll keep this issue updated with any decisions, of course.

**Comment by amandavarella** [comment]
> This shows how long it's been since I worked on this project... this feature was already partially implemented in #14, by adding a "Clear all votes from this list" menu item to the list menu (i.e. by "column", @henry-pozzetta).

That will do! Thanks for that
